src/main/java/org/fundacionjala/at15/katas/langtonant/adrian/Ant.java

```diff
@@ -0,0 +1,82 @@
+package org.fundacionjala.at15.katas.langtonant.adrian;
+
+public class Ant {
+    private int valueinx;
+    private int valueiny;
+    private int state;
+    private final int up = 0;
+    private final int right = 1;
+    private final int down = 2;
+    private final int left = 3;
+    private int direction;
+    private Board tablero;
```

Comment:
We need to switch to english this one as well. Make sure to check all the code for this, I think there is something similar in the Main class